manticore/platforms/evm.py

```diff
@@ -2553,10 +2552,12 @@ def get_storage_data(self, storage_address, offset):
         :return: the value
         :rtype: int or BitVec
         """
-        value = self._world_state[storage_address]["storage"].get(offset, 0)
+        value = self._world_state.get_storage_data(storage_address, offset)
```

Comment:
I think this behavior should be preserved. `get_storage_data` will be called on the overlay. which will in turn call the underlay here:
https://github.com/trailofbits/manticore/blob/06da949259349ed847f213abe23bd56c65d237d5/manticore/platforms/evm_world_state.py#L346
If the underlay is a `DefaultWorldState`, it will return 0:
https://github.com/trailofbits/manticore/blob/06da949259349ed847f213abe23bd56c65d237d5/manticore/platforms/evm_world_state.py#L141-L142